www/components/gallery/Carousel.tsx

```diff
@@ -1,3 +1,4 @@
+import "$fresh/src/server/constants.ts";
```

Comment:
I think this line should be at the top of where `asset()` is defined, `/src/runtime/utils.ts`.

Reply:
I have tried but it seems that change causes some errors when testing the fresh module